grpc/runtime/src/main/scala/io/buoyant/grpc/runtime/H2Headers.scala

```diff
@@ -0,0 +1,21 @@
+package io.buoyant.grpc.runtime
+
+import com.twitter.finagle.buoyant.h2
+
+private[runtime] object H2Headers {
+  private[this] val ContentType = "content-type"
```

Comment:
we can reuse the constant defined at `com.twitter.finagle.http.Fields.ContentType`

Reply:
I'd have to adjust dependencies for this to work. IMO, not worth it for a single string constant. But let me know please if you believe it's better that way, I'll update deps and code then

Reply:
fair enough!